examples/material-designer-workflow/main.py

```diff
@@ -0,0 +1,71 @@
+# # Material Designer
+
+# This notebook demonstrates the process of sending user-defined parameter values to a parameterized analysis and receive the corresponding simulation output via a Workbench service on a local machine. 
+
+import os
+import pathlib
+
+from ansys.workbench.core import launch_workbench
+
+# Launch the Workbench service on the local machine, using some options. 
+# Define several directories that will be used during the session. 
+# `workdir` is set to the parent directory of the current file. 
+# `assets`, `scripts`, and `wbpz` are subdirectories within the working directory. 
+# The `launch_workbench` function is called to start a Workbench session with specified directory.
+
+workdir = pathlib.Path("__file__").parent
+
+assets = workdir / "assets"
+
+wb = launch_workbench(client_workdir=str(workdir.absolute()))
+
+# Upload the project files to the server using the `upload_file_from_example_repo` method. 
+# The file to upload is `MatDesigner.wbpz`.
+
+wb.upload_file_from_example_repo(dirname="material-designer-workflow/wbpz",filename='MatDesigner.wbpz')
+
+# Execute a Workbench script (`project.wbjn`) to define the project and load the geometry using the `run_script_file` method. 
+# The `set_log_file` method is used to direct the logs to `wb_log_file.log`.
+
+export_path = 'wb_log_file.log'
+wb.set_log_file(export_path)
+sys_name = wb.run_script_file(str((assets / "project.wbjn").absolute()), log_level='info')
+
+# Prepare the Workbench command template to make modifications to the material property, in this case the Young's modulus of the material
+
+wbjn_template = """designPoint1 = Parameters.GetDesignPoint(Name="0")
+parameter1 = Parameters.GetParameter(Name="P1")
+designPoint1.SetParameterExpression(
+    Parameter=parameter1,
+    Expression="{} [Pa]")
+backgroundSession1 = UpdateAllDesignPoints(DesignPoints=[designPoint1])
+"""
+
+# Update the project with a new value for the Young's modulus
+
+my_command = wbjn_template.format( 1.6e10 )
```

Comment:
A nice way to pass an input to WB script!  @vnamdeo You asked for how to do this the other day.